packages/chat-component/src/main.ts

```diff
@@ -161,6 +176,12 @@ export class ChatComponent extends LitElement {
         this.isResetInput = false;
 
         const response = this.apiResponse as BotResponse;
+        // adds thought process support when streaming is disabled
```

Comment:
Why only when it's disabled? It's also available when streaming is enabled on the API (it comes in the first chunk)

Reply:
The logic is different because when streaming is enabled, data_points and thoughts are part of a different chunk of the streamed response. See line 98 for streamed enabled support.